### PR TITLE
Update PeopleConverterJpaServiceImpl.java

### PR DESCRIPTION
**Pull request by JellyBoonz**
Merged the two if statements. Formatting changes to reduce code smell. No change in logic/behavior.

### CODE REVIEW

**Review by NyorJa — CHANGES_REQUESTED** [review]
I suggest use apache commons lang3 StringUtils.equals()

It has null checking and comparison

**Comment by JellyBoonz** [comment]
I made changes to use StringUtils.equals(). Let me know if this is sufficient.

File: src/main/java/org/trebol/jpa/services/conversion/PeopleConverterJpaServiceImpl.java [hunk 1]
```diff
@@ -72,17 +72,13 @@ public Person applyChangesToExistingEntity(PersonPojo source, Person existing) t
 
     // phones may be empty, but not null
     String phone1 = source.getPhone1();
-    if (phone1 != null) {
-      if (!target.getPhone1().equals(phone1)) {
+    if (!phone1.equals(null) && !target.getPhone1().equals(phone1)) {
```
Comment (NyorJa) [inline src/main/java/org/trebol/jpa/services/conversion/PeopleConverterJpaServiceImpl.java:75]: Use apache commons lang 3..

`!StringUtils.equals(phone1, target.getPhone1())`

**Comment by bglamadrid** [comment]
Sorry @JellyBoonz, PR #196 has this covered in full. I'll take that one. 
Still thank you very much. You can totally go for another good first issue like this if you want to 👍

**Pull request closed without merging**